### ISSUE TITLE
About variable e24516 in the puf.csv file

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/Tax-Calculator#738]
Situation: The `e24516` variable is in the `puf.csv` file and it is "aged" in the Records class `_blowup` method, but it is never used in the `functions.py` calculations.  Here are the grep results over the whole Tax-Calculator source tree:
```
tax-calculator$ grep -r --include="*.py" e24516 .
./taxcalc/records.py:        'p23250', 'e24515', 'e24516', 'e24518',
./taxcalc/records.py:        self.e24516 *= ACGNS
```

Question: Is it a mistake that `e24516` is never used in the calculations, or is `e24516` redundant, and therefore, should be removed from the `puf.csv` file?

@MattHJensen @feenberg @Amy-Xu @GoFroggyRun

**Comment by GoFroggyRun** [comment]
@martinholmer 
In the 09 puf booklet, the variable `e24516 ` is documented as 'Capital gain less investment expense', which deals with capital gains. 

After looking at the tax logic, it seems to me that this variable won't be used at all, since we readily have `c24516`, which is calculated using other variables. 

And thus this variable does seem redundant to me at this moment.

**Comment by martinholmer** [comment]
@GoFroggyRun,  Thanks for your feedback.  So, its one vote for redundant.
Anybody else want to vote?  @MattHJensen @feenberg @Amy-Xu

**Comment by martinholmer** [comment]
Issue #738, which is now being closed, is resolved by pull request #740.
